Adding the file with names, while I figure out Jekyll (#146)
open-life-science/open-life-science.github.io: _posts/2020-05-27-ols1-wrapup.md

---
layout: post
title: End of OLS-1 and OLS-2 Announcement post
authors: 
- yochannah
- bebatut
- malvikasharan
image: https://images.unsplash.com/photo-1577036421869-7c8d388d2123?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80

> ---

Reviewer: ```suggestion
photos:
  name: 
  license: 
  url: https://images.unsplash.com/photo-1577036421869-7c8d388d2123?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80
---
```

Reviewer: Do you have the details for the details about the picture?

> 

Reviewer: ```suggestion
{% assign people = site.data.people %}
{% assign projects = site.data.ols-1-projects %}

{% assign grad-participants = '' %}
{% assign grad-mentors = '' %}
{% assign grad-projects = '' %}
{% assign all-participants = '' %}
{% assign all-mentors = '' %}
{% for project in projects %}
    {% if project.graduated %}
        {% capture grad-projects %}{{ grad-projects}}, {{ project.name }}{% endcapture %}
        {% for p in project.participants %}
            {% capture grad-participants %}{{ grad-participants}}, [{{ people[p].first-name }} {{ people[p].last-name }}](/ols-1/projects-participants#{{ p }}){% endcapture %}
        {% endfor %}
        {% for m in project.mentors %}
            {% capture grad-mentors %}{{ grad-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/ols-1#{{ m }}){% endcapture %}
        {% endfor %}
    {% endif %}
    {% for p in project.participants %}    
        {% capture all-participants %}{{ all-participants}}, [{{ people[p].first-name }} {{ people[p].last-name }}](/ols-1/projects-participants#{{ p }}){% endcapture %}
    {% endfor %}
    {% for m in project.mentors %}
        {% capture all-mentors %}{{ all-mentors }}, [{{ people[m].first-name }} {{ people[m].last-name }}](/ols-1#{{ m }}){% endcapture %}
    {% endfor %}
{% endfor %}

{% assign grad-projects = grad-projects | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign grad-participants = grad-participants | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign grad-mentors = grad-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign all-participants = all-participants | remove_first: ', ' | split: ", " | uniq | sort %}
{% assign all-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
```

Reviewer: To collect lists of graduated projects, people, mentors and full lists too

Author: Thank you @bebatut. It would have taken me hours to add that.

> *We are extremely delighted to announce that leads to 18 projects from our first cohort have graduated from the Open Life Science program.

Reviewer: ```suggestion
*We are extremely delighted to announce that leads to {{ grad-projects | size }} projects from our first cohort have graduated from the Open Life Science program.
```

Author: I think this will give 20 projects but in the end 2 people never finished. We might need to add a rule there?

Author: I see that you have added grad* tags <3

We held 5 separate graduation ceremonies on 17 April, 20 May, and 27 May 2020 in different time zones. In their final presentations, each team/individual member amazed us with the progress they made in their project and the level of dedication and resilience they showed in leading their work, especially during the pandemic.*

> We congratulate all our proud mentees: [Anunya Opasawatchai](https://openlifesci.org/ols-1/projects-participants#anayan321), [Billy (William) Broderick](https://openlifesci.org/ols-1/projects-participants#billbrod), [Bruno Eleres Soares](https://openlifesci.org/ols-1/projects-participants#bruno-soares), [Cassandra Gould van Praag](https://openlifesci.org/ols-1/projects-participants#cassgvp), [Chiara Bertipaglia](https://openlifesci.org/ols-1/projects-participants#ChiaraBertipaglia), [Christine Rogers](https://openlifesci.org/ols-1/projects-participants#christinerogers), [David Chentanaman](https://openlifesci.org/ols-1/projects-participants#davidviryachen), [David Kiragu Mwaura](https://openlifesci.org/ols-1/projects-participants#Kiragu-Mwaura), [David LeBauer](https://openlifesci.org/ols-1/projects-participants#dlebauer), [Elsa Loissel](https://openlifesci.org/ols-1/projects-participants#elsasci), [Emily Cain](https://openlifesci.org/ols-1/projects-participants#MagicMilly), [Festus Nyasimi](https://openlifesci.org/ols-1/projects-participants#Fnyasimi), [Kristina Riemer](https://openlifesci.org/ols-1/projects-participants#KristinaRiemer), [Lena Karvovskaya](https://openlifesci.org/ols-1/projects-participants#LangData), [Lilian Juma](https://openlifesci.org/ols-1/projects-participants#Lilian9), [Margaret Wanjiku](https://openlifesci.org/ols-1/projects-participants#Megmugure), [Matúš Kalaš](https://openlifesci.org/ols-1/projects-participants#matuskalas), [Michael Kofia Landi](https://openlifesci.org/ols-1/projects-participants#LandiMi2), [Naraiana Loureiro Benone](https://openlifesci.org/ols-1/projects-participants#NaraLB), [Renato Alves](https://openlifesci.org/ols-1/projects-participants#unode), [Samuel Guay](https://openlifesci.org/ols-1/projects-participants#SamGuay), [Sandy Kawano](https://openlifesci.org/ols-1/projects-participants#MorphoFun), [Sudarshan GC](https://openlifesci.org/ols-1/projects-participants#sudarshangc), and [Zoe Chervontseva](https://openlifesci.org/ols-1/projects-participants#homo-sapiens34).

Reviewer: ```suggestion
We congratulate all our proud mentees: {{ grad-participants | join: ', ' }}.
```

Author: Wooo - thanks a lot for sorting this!

> We are very grateful to our mentors for setting such a high standard of mentorship that positively impacted the overall success of this cohort. Thank you, [Andrew Stewart](https://openlifesci.org/ols-1#ajstewartlang), [Aidan Budd](https://openlifesci.org/ols-1#aidanbudd), [Björn Grüning](https://openlifesci.org/ols-1#bgruening), [Caleb Kibet](https://openlifesci.org/ols-1#kipkurui), [Daniela Saderi](https://openlifesci.org/ols-1#dasaderi), [Fotis Psomopoulos](https://openlifesci.org/ols-1#fpsom), [Hao Ye](https://openlifesci.org/ols-1#ha0ye), [Harry Smith](https://openlifesci.org/ols-1#Zebetus), [Holger Dinkel](https://openlifesci.org/ols-1#hdinkel), [Katrin Leinweber](https://openlifesci.org/ols-1#katrinleinweber), [Luis Pedro Coelho](https://openlifesci.org/ols-1#luispedro), [Malvika Sharan](https://openlifesci.org/ols-1#malvikasharan), [Mateusz Kuzak](https://openlifesci.org/ols-1#mkuzak), [Naomi Penfold](https://openlifesci.org/ols-1#npscience), [Patricia Herterich](https://openlifesci.org/ols-1#pherterich) , [Renato Alves](https://openlifesci.org/ols-1#unode), [Rodrigo Oliveira Campos](https://openlifesci.org/ols-1#rodrigocam), [Toby Hodges](https://openlifesci.org/ols-1#tobyhodges), [Vicky Nembaware](https://openlifesci.org/ols-1#VickyNembaware), and [Yo Yehudi](https://openlifesci.org/ols-1#yochannah).

Reviewer: ```suggestion
We are very grateful to our mentors for setting such a high standard of mentorship that positively impacted the overall success of this cohort. Thank you, {{ grad-mentors | join: ', ' }}.
```

Last but not least, thanks to our expert community and speakers for sharing your expertise and personal stories that were educational and inspiring for all of us.

## Presentations - recap and common themes

> Let us recap it for you - we ran our first cohort from January to May 2020 with 29 mentees, 20 mentors, and over 50 experts from Asian, African, European, Latin American, North American, and Russian countries.

Reviewer: ```suggestion
Let us recap it for you - we ran our first cohort from January to May 2020 with {{ all-participants | size }} mentees, {{ all-mentors | size }} mentors, and over 50 experts from Asian, African, European, Latin American, North American, and Russian countries.
```

> Our program introduced our mentee to a great deal of Open Science concepts that are directly applicable to their work. In addition, they also learned all about designing their project for inclusivity, establishing connections with others in the program, managing their communities, and working collaboratively. 

Reviewer 2: ```suggestion
Our program introduced our mentees to a great deal of Open Science concepts that are directly applicable to their work. In addition, they also learned all about designing their project for inclusivity, establishing connections with others in the program, managing their communities, and working collaboratively. 
```

Through this program, OLS helped design and implement projects to solve technical and cultural questions by collaboratively designing open source software, AI techniques, educational resources, culture movements, and global partnerships in their communities (detail: https://openlifesci.org/ols-1/projects-participants/). 

The final step to graduate Open Life Science is a short presentation about what you worked on. While we originally envisioned this being in April, COVID-19 brought a whole new set of pressures, so we decided to extend the deadlines for final presentations, as well as offering additional timeslots. The result - five calls - can all be viewed on [our YouTube channel](https://www.youtube.com/playlist?list=PL1CvC6Ez54KB6U9GtjOjwESMurHgT41qM&advanced_settings=1&disable_polymer=1)
 
Some of the common themes that surprised us and brought us joy during the presentations: 

> The COVID-19 pandemic had an unexpected impact of our program beyond the initial goals as our mentees could transfer their OLS skills in other areas like establishing effective communication among the international COVID collaborations, designing the format of their lab meetings online, setting mentor-mentee relationships in their local networks, and connecting unemployed folk musicians and artists with each other in Nepal during the lockdown.

Reviewer 2: ```suggestion
- The COVID-19 pandemic had an unexpected impact of our program beyond the initial goals as our mentees could transfer their OLS skills in other areas like establishing effective communication among the international COVID collaborations, designing the format of their lab meetings online, setting mentor-mentee relationships in their local networks, and connecting unemployed folk musicians and artists with each other in Nepal during the lockdown.
```

Reviewer 2: adds bullet points

> The [open canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit#slide=id.p) exercise was commonly useful for everyone. We presented this in [our first cohort call](https://openlifesci.org/ols-1/week02/), but a large number of mentees found it so foundational they shared it even 3-4 months later in their presentations. 

Reviewer 2: ```suggestion
- The [open canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit#slide=id.p) exercise was commonly useful for everyone. We presented this in [our first cohort call](https://openlifesci.org/ols-1/week02/), but a large number of mentees found it so foundational they shared it even 3-4 months later in their presentations. 
```

> Skills like collaborative work culture, self care, ally skills, giving and receiving feedback, agile management, and knowledge dissemination are valuable beyond research.

Reviewer 2: ```suggestion
- Skills like collaborative work culture, self care, ally skills, giving and receiving feedback, agile management, and knowledge dissemination are valuable beyond research.  
```

A Youtube Stream can make all the difference in helping your friends and family attend important events: 

> <blockquote class="twitter-tweet"><p lang="en" dir="ltr">The graduation was even more special because my <a href="https://twitter.com/hashtag/VUamsterdam?src=hash&amp;ref_src=twsrc%5Etfw">#VUamsterdam</a> family, <a href="https://twitter.com/gravana?ref_src=twsrc%5Etfw">@gravana</a>, and <a href="https://twitter.com/PhDToothFAIRy?ref_src=twsrc%5Etfw">@PhDToothFAIRy</a><br>were there 😍<br>AND, my mother watched my graduation in real-time. She had to miss both my wedding and PhD defense. It&#39;s the first event in my life abroad that she witnessed 👩‍👧 <a href="https://t.co/xWWaJ9kLhw">pic.twitter.com/xWWaJ9kLhw</a></p>&mdash; Lena Karvovskaya (@LangData) <a href="https://twitter.com/LangData/status/1263468708330078209?ref_src=twsrc%5Etfw">May 21, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 

Reviewer: I could change a bit the CSS to center it. What do you think?

Author: Sounds great!

## Announcing OLS-2 - please share with your networks! 

Motivated by the overwhelming success of OLS so far, we’re incredibly excited to announce that applications for round 2 of OLS are now officially open!  
Whether you were excited by OLS-1 but missed the deadline, or whether you’ve heard about it for the first time today, you’re welcome to apply. 

> To apply, please visit [EasyChair](https://easychair.org/my/conference?conf=ols2). We also have [templates](https://github.com/open-life-science/application-forms) you can download to use when preparing your application. 

Reviewer: We could use here the timeline as on the index page:

![Screenshot 2020-05-28 at 10 04 12](https://user-images.githubusercontent.com/1842467/83115426-a167e500-a0ca-11ea-8e24-a3db0577fd32.png)

What do you think?

It can be done with `{% include _includes/ols-2-timeline.md %}`

> Questions, comments, looking for more advice? You’re always welcome to email the OLS leadership at hello@openlifesci.org - and we have two pre-application webinars scheduled: 

Reviewer: ```suggestion
Questions, comments, looking for more advice? You’re always welcome to email the OLS leadership at [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }})- and we have two pre-application webinars scheduled: 
```

- bebatut
- malvikasharan
image: https://images.unsplash.com/photo-1577036421869-7c8d388d2123?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80
photos:

>   name: 
>   license: 

Author: ```suggestion
  name: Photo on Unsplash by @claybanks
  license: CC-BY
```

Some of the common themes that surprised us and brought us joy during the presentations: 

> - The COVID-19 pandemic had an unexpected impact of our program beyond the initial goals as our mentees could transfer their OLS skills in other areas like establishing effective communication among the international COVID collaborations, designing the format of their lab meetings online, setting mentor-mentee relationships in their local networks, and connecting unemployed folk musicians and artists with each other in Nepal during the lockdown.

Author: ```suggestion
- The COVID-19 pandemic had an unexpected impact of our program beyond the initial goals as our mentees could transfer their OLS skills in other areas like establishing effective communication among the international COVID collaborations, designing the format of their lab meetings online, setting mentor-mentee relationships in their local networks, and connecting folk musicians and artists with each other in Nepal during the lockdown.
```
